services/openshift/openshift.service

```diff
@@ -11,8 +11,7 @@ Restart=always
 EnvironmentFile=-/etc/sysconfig/openshift_option
 ExecStartPre=-/usr/bin/docker stop openshift
 ExecStartPre=-/usr/bin/docker rm openshift
-ExecStartPre=-/usr/bin/docker tag ${DOCKER_REGISTRY}/${IMAGE} openshift
-ExecStartPre=-/usr/bin/sh /opt/adb/openshift/docker_config
+ExecStartPre=-/usr/bin/sh /opt/adb/openshift/docker_config $OPTIONS
```

Comment:
is `docker_config $OPTIONS` inherited from old code or we are adding $options as argument?